app/.idea/gradle.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would you mind not adding these 3 .idea files? You can actually add them to `.gitignore`. Thanks!